### PR TITLE
Setup Rails Project and Add Schema

### PR DESCRIPTION
**Pull request by sharon-odhiambo**
- In this PR I created the Blog App Data Model and Added the migration project schema.

**Below are the highlights:**

- Create new Blog App
- Setup the project with postgres database
- Build project schema.
- Create and run the necessary migration files.
- Created tables that match the ERD diagram.

Cheers and Happy Reviewing!

### CODE REVIEW

**Review by FrederickMih — COMMENTED** [review]
Hi @sharon-odhiambo , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not contain a descriptive README.md file.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by sharon-odhiambo** [comment]
-Hi @FrederickMih , thank you so much for the review!

File: README.md [hunk 1]
```diff
@@ -1,24 +1,44 @@
-# README
+# OOP-School-Library
+- This is a school library that uses OOP to build record app for adding students, teachers, book rentals and list all the available rented books alongside either the teacher or student and indicates the date. The app organizes everything in a great way allowing easy maneuver.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Built With
+- Languages: Ruby on Rails
+- Models/Frameworks: OOP
```
Comment (julie-ify) [inline README.md]: - [ ] Kindly update your readme title and description to match the project.

- [ ] There should be `config/database.yml` file that defines the database adapter. And the adapter should use PostgreSQL. This file also defines the databases for different environments.

  Something like this: [My config/database file](https://github.com/julie-ify/Blog-app/blob/main/config/database.yml)

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @sharon-odhiambo,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sharon-odhiambo) [reply README.md]: Hi @julie-ify , thank you for the suggestions. I have implemented the changes.

**Review by DammyShittu — APPROVED** [review]
Hi @sharon-odhiambo,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sharon-odhiambo** [comment]
Hi @DammyShittu , thank you for the review.

**Pull request merged**